templates/data/custom_dataloader_plugin/custom_plugin.py

```diff
@@ -76,11 +78,12 @@ def getJobId(self):
         :returns a string
         """
         # Generate a stable id that changes with the main parameters
-        m = hashlib.sha256()
-        m.update(self.args.first_custom_param.encode('utf-8'))
-        m.update(self.args.second_custom_param.encode('utf-8'))
+        m = hashlib.blake2b(digest_size=20)
```

Comment:
Just curious: why blake2b?

Reply:
Took it from here: https://github.com/squirro/dataloader-plugins/blob/master/onedrive_plugin/dataloader_plugin.py#L306

Wikipedia claims blake is both faster and more secure than the SHA's, but it doesn't matter in the end.